src/OpenDebugAD7/MILaunchOptions.cs

```diff
@@ -292,12 +292,28 @@ private static string CreateArgumentList(IEnumerable<string> args)
                     if (stringBuilder.Length != 0)
                         stringBuilder.Append(' ');
 
-                    stringBuilder.Append(Terminal.Quote(arg));
+                    stringBuilder.Append(QuoteArgument(arg));
                 }
             }
             return stringBuilder.ToString();
         }
 
+        // gdb does not like parenthesis without being quoted
+        private static char[] s_CHARS_TO_QUOTE = { ' ', '\t', '(', ')' };
```

Comment:
@pieandcakes shouldn't quote be in this list too?

Reply:
Technically because we are replacing the quote with the escaped quote, it isn't necessary. I tested it and it doesn't seem to be an issue.